### DISCUSSION TITLE
New, shiny, app & document icons

### DISCUSSION THREAD

**Discussion by Zabriskije** [transmission/transmission#5479] [Show and tell]
Hi folks,
The current app icon — while being well made — looks a bit old school to me. I had in mind to do a more modern one for a long time, and two days ago, I finally did it:

<img width="128" src="https://user-images.githubusercontent.com/101254295/235318422-903dfddb-ee15-4f4b-adf3-056e19a9115e.png">

<img width="128" src="https://user-images.githubusercontent.com/101254295/235318445-fc15ed55-ab73-4f29-adb7-f509b8095448.png">

I'd like to propose it as the new icon.
[Zip with PNG/ICNS/Iconset files](https://github.com/transmission/transmission/files/11359856/Transmission.zip)

**Comment by dantearmok** [comment]
Nice icons!  I like 'em a lot!

**Comment by arcticfoxwp** [comment]
How do we go about getting this added as the new app icon?

**Reply by Zabriskije** [reply]
If the project was early or a small one, I would have made a PR directly; but when the project is that big, normally devs and contributors are attached to the logo, icons, and visuals in general (can't blame them) — in this case, I prefer to get feedback first.

We just have to wait and hope people leave a like or a comment so the discussion gets traction 🤞

**Reply by nevack** [reply]
As a side note - we don't need Document icons for newer macOS version as system can generate them uniformly from App Icon and extension.
This way document icons can get some updated style in future basically for free :)

**Reply by Zabriskije** [reply]
Oh, I know, but it's cooler without the icon background 😎

**Comment by GaryElshaw** [comment]
Just my 2c, but i think these would look more at home on iOS than macOS.

**Reply by Zabriskije** [reply]
Is this because it's a bit more flat? If that's the case, I did it on purpose since that's what I prefer, and it's the general trend starting from Big Sur. Many apps have switched to this style, so it helps with visual consistency.

Example (1st image proposed icon, 2nd image current icon):

<img width="390" src="https://github.com/transmission/transmission/assets/101254295/f7eb4880-05a9-4f33-ae71-c950b2c0a7b9"> <img width="390" src="https://github.com/transmission/transmission/assets/101254295/bc1d7adb-c260-47d9-82bf-e77455aaeb3f">

It's still recognizable and maintains its character while integrating better with the general aesthetic.

**Comment by arcticfoxwp** [comment]
@GaryElshaw As shown, this new icon fits MacOS styling better than the current icon.

**Reply by GaryElshaw** [reply]
As i said, just my 2c. To me, and no shade on Zabriskije's efforts at all, the icon looks more in keeping with iOS. If i could change the current macOS icon, it would probably be reduced to a greater level of abstraction and resemble, from the examples above, Codium, Transmit, or Hack.

![arrow up and down square@100x](https://github.com/transmission/transmission/assets/69029666/41cd1388-e2ec-4fcb-a41b-d998df5de0e9)

![image](https://github.com/transmission/transmission/assets/69029666/acc33bd3-c4e9-4053-88da-f8c4db17f912)

![image](https://github.com/transmission/transmission/assets/69029666/87c623ff-0beb-4ed8-929b-a7a154d38dbd)

**Reply by arcticfoxwp** [reply]
@GaryElshaw I'm not sure you really know what you are talking about design-wise. Are you using an older version of MacOS / iOS or something?

All of Apple's apps (and increasingly so from developers) use the same icon for MacOS and iOS.

ex. App Store, Messages, Music, Photos, Notes, Home, etc. In fact, of all the apps I use, I can't even find one that uses a different icon for each system.

If you think the current Transmission icon,  which is many years old now, is better than the one proposed here, then I question the design ability of the dev team.

Anyway, nice work @Zabriskije, I hope the dev team gets it together here.

**Reply by nevack** [reply]
> If you think the current Transmission icon, which is many years old now

How so? :) Just 2 years ago 

https://github.com/transmission/transmission/issues/1360
https://github.com/transmission/transmission/pull/1650
